src/daft-catalog/src/bindings.rs

```diff
@@ -111,6 +111,11 @@ impl<T> Bindings<T> {
     pub fn is_empty(&self) -> bool {
         self.bindings.is_empty()
     }
+
+    /// Returns the binding if it exists (exact-case).
+    pub fn get_exact(&self, name: &str) -> Option<&T> {
+        self.bindings.get(name)
+    }
```

Comment:
I don't believe we want to expose this since lookups require giving the casing mode so we don't introduce inconsistent name resolution bugs.

Reply:
yeah this was just kinda hacked in here so i could work on the udf logic. Was planning on coming back to this and use `lookup`